pennylane_lightning/lightning_gpu/_measurements.py

```diff
@@ -138,6 +145,64 @@ def _process_single_shot(samples):
             tuple(zip(*processed_samples)) if shots.has_partitioned_shots else processed_samples[0]
         )
 
+    def expval(self, measurementprocess: MeasurementProcess):
```

Comment:
For now, It was necessary to include the base methods 'expval' because differ a lot from the base class. Let us leave here and in the further PR I will try to find a better solution.

Reply:
Could you create a single SC ticket for revisiting LGPU expval, jacobian, jvp, ... in the backlog? I'd be happy to get back to this as a follow-up.

Reply:
Done! [sc-74360]